src/scraper.rs

```diff
@@ -253,6 +270,8 @@ mod tests {
             user_agent: "suckit".to_string(),
             random_range: 5,
             verbose: true,
+            include: "mp[3-4]".to_string(),
+            exclude: "mp[3-4]".to_string(),
```

Comment:
Just reuse the same value for `include` and `exclude` in `test_zero_delay_range` and `test_non_zero_delay_range` to avoid confusion

Reply:
✔️